applications/portfolioDrafts/getPortfolioDrafts.ts

```diff
@@ -0,0 +1,61 @@
+import { ScanCommand, ScanCommandInput } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { ErrorCodes } from "./models/Error";
+import { PortfolioDraftSummary } from "./models/PortfolioDraftSummary";
+import { dynamodbDocumentClient as client } from "./utils/dynamodb";
+import { ApiSuccessResponse, ErrorResponse, ErrorStatusCode, SuccessStatusCode } from "./utils/response";
+
+const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+const QUERY_PARAM_INVALID = new ErrorResponse(
+  { code: ErrorCodes.INVALID_INPUT, message: "Invalid request parameter" },
+  ErrorStatusCode.BAD_REQUEST
+);
+
+/**
+ * Evaluate query string parameter which is expected to be an integer.
+ * Return defaultInt if undefined or NaN.
+ * @param qparam - The query string parameter to evaluate
+ * @param defaultInt - The default value
+ */
+function evaluateQueryParameterInteger(qparam: string | undefined, defaultInt: number): number {
+  // assert numeric
+  if (!qparam?.match(/^\+?([0-9]\d*)$/)) {
```

Comment:
```suggestion
  if (!qparam?.match(/^\d+$/)) {
```

simplifies this to accept any string of 0 or more digits (`0-9`). it will _reject_ a leading `+`.

Reply:
That's nice.

Reply:
@zachclark-ccpo Just want to highlight this could be something really good to get unit tests for early on as well, if you need something as a good place to get started. Especially if it got pulled into `utils/` somewhere and `export`ed. But if this is the only place we'll ever end up needing to do this, then maybe it's not appropriate for `utils/`. Something to consider.